src/stories/containers/RecognizedDelegates/KeyStats/KeyDescriptionCards/PercentageTotalCard.tsx

```diff
@@ -0,0 +1,42 @@
+import styled from '@emotion/styled';
+import React from 'react';
+import KeyStatsCard from '../../GenericDelegateCard';
+import { DescriptionDelegates } from './TotalRecognizedDelegatesCard';
+
+interface Props {
+  percent: string;
+}
+
+const PercentageTotalCard: React.FC<Props> = ({ percent }) => (
+  <ExtendedGenericDelegate>
+    <Annual>{percent}</Annual>
```

Comment:
The `percent` should be a number and this component should be the one responsible to format it, so if the format changes, you don't have to follow the track of where this is formatted every time the component is used. If it is needed you can create a helper function in the `utils` folder in order to reduce the amount of logic used here and allow to reuse that function later (if it does not exist already)